recipe/0001-inject-cmake-project.patch

```diff
@@ -88,34 +91,34 @@ index 0000000..c15df47
 +# allow getopt to work with find_package
 +export(EXPORT getoptTargets
 +  NAMESPACE unofficial::getopt-win32
-+  FILE "${CMAKE_CURRENT_BINARY_DIR}/getoptTargets.cmake"
++  FILE "${CMAKE_CURRENT_BINARY_DIR}/unofficial-getopt-win32Targets.cmake"
 +)
 +
 +install(EXPORT getoptTargets
 +  NAMESPACE unofficial::getopt-win32
-+  FILE getoptTargets.cmake
-+  DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/getopt
++  FILE unofficial-getopt-win32Targets.cmake
++  DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/unofficial-getopt-win32
 +)
 +
 +include(CMakePackageConfigHelpers)
 +
 +configure_package_config_file(${CMAKE_CURRENT_SOURCE_DIR}/Config.cmake.in
-+  "${CMAKE_CURRENT_BINARY_DIR}/getoptConfig.cmake"
-+  INSTALL_DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/getopt
++  "${CMAKE_CURRENT_BINARY_DIR}/unofficial-getopt-win32Config.cmake"
++  INSTALL_DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/unofficial-getopt-win32
 +  NO_SET_AND_CHECK_MACRO
 +  NO_CHECK_REQUIRED_COMPONENTS_MACRO
 +  )
 +
 +write_basic_package_version_file(
-+  "${CMAKE_CURRENT_BINARY_DIR}/getoptConfigVersion.cmake"
++  "${CMAKE_CURRENT_BINARY_DIR}/unofficial-getopt-win32ConfigVersion.cmake"
 +  VERSION "${PROJECT_VERSION}"
 +  COMPATIBILITY AnyNewerVersion
 +  )
 +
 +install(FILES
-+  ${CMAKE_CURRENT_BINARY_DIR}/getoptConfig.cmake
-+  ${CMAKE_CURRENT_BINARY_DIR}/getoptConfigVersion.cmake
-+  DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/getopt
++  ${CMAKE_CURRENT_BINARY_DIR}/unofficial-getopt-win32Config.cmake
++  ${CMAKE_CURRENT_BINARY_DIR}/unofficial-getopt-win32ConfigVersion.cmake
++  DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/unofficial-getopt-win32
```

Comment:
Why use the term 'unofficial' instead of maybe 'conda-forge'?

Reply:
This is emulating the behavior of the package when it gets installed with vcpkg. Where you `find_package(unofficial-getopt-win32)`, and people always appreciate consistency, even if it is a bit obtuse.

Reply:
I'd be happy to change it to `conda-forge` but there is precedence for the `unofficial` name. Let me know your decision.

Reply:
Please add a comment explaining this at the top of the file.

Reply:
done